backend/routes/verifiedusers.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
+
+const Router = require('koa-router');
+const parser = require('koa-body');
+
+const Users = require('../models/users');
+
+const parsers = {
+    query: parser({urlencoded: true, multipart: false, json: false}),
+    form: parser({urlencoded: false, multipart: true, json: false}),
+    json: parser({urlencoded: false, multipart: false, json: true}),
+};
+
+const verifiedUsers = new Router();
+
+verifiedUsers.get('/', async ctx => {
+   let response = {};
+    try {
+        let verifiedUsers = await Users.query()
+            .select('id')
+            .where('is_verified', true)
+        response.result = 'success';
+        response.verifiedIds = idHash(verifiedUsers)
```

Comment:
I think with `_.filter` it could be inlined, let's be elegant. :) Also we could use the native `filter()` method, but since we have lodash anyway..

Reply:
Want the ids to all be in a hash for quick lookup when comparing against reporterIds in the frontend, will change if you know how to do that with lodash!